tony-core/src/main/resources/tony-default.xml

```diff
@@ -202,4 +202,13 @@
     <name>tony.application.yarn-conf-path</name>
   </property>
 
+  <!-- History folder configurations -->
+  <property>
+    <description>
+      Location for Tony AM to write config and events on HDFS
+    </description>
+    <name>tony.history.location</name>
+    <value>/system/tony-history/</value>
```

Comment:
I prefer leaving off the final `/` (so `/system/tony-history` rather than `/system/tony-history/`). If you look at other path-related configs in `$HADOOP_CONF_DIR`, you'll see that most leave off the final `/`.

Reply:
Ok sure. Fixed: https://github.com/linkedin/TonY/pull/79/commits/e489758601fce6c775bd5d11e0f97289e0664a81

Reply:
Ok, sure. Fixed: https://github.com/linkedin/TonY/pull/79/commits/24b15e952b42cfafb8ed4d3a2bc72afb4fe3e185